server/lib/imageUrlToAmazonUrl.js

```diff
@@ -15,6 +15,10 @@ import MultiPartUpload from 'knox-mpu-alt';
 *     @param aws_src {String}
 */
 function imageUrlToAmazonUrl(knox_client, src, callback) {
+  // we skip this in development environment since we don't have the AWS credentials
```

Comment:
I'd vote to minimize or eliminate dev vs prod checks. Historically, this function has been somewhat unstable. And disabling it in development means less likely for us to experience the pain ourselves.

Maybe better to check that if aws keys exist, then attempt to upload? (My dev environment keeps aws keys so it'll keep working).